netlify/README.md

```diff
@@ -0,0 +1,13 @@
+# Netlify
+
+Purpose: Provides a platform for the development and maintenance of the spiffe.io website, including CDN, automated deploys, HTTPS etc.
+
+Primary Owner: @lucperkins, @caniszczyk, @celestehorgan, Unknown
+
+Users/Groups with Access: @mchurichi, @drrt, @ajessup, @sanderson042, Unknown
+
+Billing: Unknown
```

Comment:
Any idea if the owners and access lines here are correct, and whether or not we are paying for Netlify? Sanity check on the Purpose would be good too... I see that spiffe.io nameservers are at NS1, is that handled by Netlify or is it separate?

Reply:
The website uses Netlify's free tier, and the NS1 nameservers are on behalf of Netlify. Needed to do that to use Netlify's LetsEncrypt functionality (instead of fronting it with Cloudflare)

Reply:
Ok great, good to know. I will update this line.

Any idea if the `Users/Groups with Access` field is complete? If so, I'll remove the `Unknown` element

Reply:
Looks like it's not entirely correct, here's the definitive list:

Owner: Zachary Corleissen
Owner: Chris Aniszczyk
Owner: Celeste Horgan
Owner: Maximiliano Churichi
Owner: David Langhorst
Owner: Andres Vega
Collaborator: Andrew J
Collaborator: Steve Anderson